src/lib/utils/auth-data.ts

```diff
@@ -0,0 +1,10 @@
+import { saleorApp } from "../../saleor-app";
+import { env } from "@/lib/env.mjs";
+
+export const getAuthData = async () => {
+  const { apl } = saleorApp;
+  // ! env.SALEOR_API_URL is not required but it is type casted here
```

Comment:
So if wont be provided, the app will fail silently.

SALEOR_API_URL should be then mandatory, or the code using this function should handle error.